src/Menu/Matcher/Voter/AdminVoter.php

```diff
@@ -42,11 +69,16 @@ public function matchItem(ItemInterface $item)
     {
         $admin = $item->getExtra('admin');
 
+        $request = $this->request;
+        if (null !== $this->requestStack) {
+            $request = $this->requestStack->getMasterRequest();
```

Comment:
Why `getMasterRequest` instead of `getCurrentRequest`?

Reply:
Well you said it on another PR:

https://github.com/KnpLabs/KnpMenu/blob/master/src/Knp/Menu/Matcher/Voter/RouteVoter.php#L59 i guess it should be master request

Reply:
https://github.com/KnpLabs/KnpMenu/blob/4c5c9be0dc94a98b51c19e148faa86830f5f8b0c/src/Knp/Menu/Matcher/Voter/RouteVoter.php#L59